extensions/Push/lib/Connector/Phabricator.pm

```diff
@@ -106,6 +106,11 @@ sub send {
               $bug->id
             ));
             make_revision_public($revision_phid);
+            # remove "secure-revision" tag
+            my $rev_obj = Bugzilla::Extension::PhabBugz::Revision->new_from_query({ phids => [ $revision_id ] });
+            my $secure_project_phid = get_project_phid('secure-revision');
+            $revision->remove_project($secure_project_phid);
+            $rev_obj->update();
```

Comment:
Also move $revision->update() to the bottom of the foreach() loop.

You may also want to create a $revision_updated boolean that you set in the if else blocks and call

$revision_obj->update() if $revision_updated;

This way we do not make an empty call to differential.revision.edit if no change was made.